independent-projects/bootstrap/maven-plugin/src/main/java/io/quarkus/maven/capabilities/CapabilityConfig.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.maven.capabilities;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CapabilityConfig {
+
+    protected String name;
+    protected List<String> onlyIf = new ArrayList<>(0);
+    protected List<String> onlyIfNot = new ArrayList<>(0);
```

Comment:
Is there a compelling reason to make these fields `protected`?

Reply:
I've made them private.